_saas-integrations/linkedin-ads/v1/linkedin-ads-v1.md

```diff
@@ -0,0 +1,98 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: LinkedIn Ads (v1.0)
+permalink: /integrations/saas/linkedin-ads/v1
+tags: [saas_integrations]
+keywords: linkedin ads, integration, schema, etl linkedin ads, linkedin ads etl, linkedin ads schema, linkedin, 
+layout: singer
+input: true
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "linkedin-ads"
+display_name: "LinkedIn Ads"
+
+singer: true 
+tap-name: "LinkedIn Ads"
+repo-url: https://github.com/singer-io/tap-linkedin-ads
+
+# this-version: "1.0"
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+status: "Open Beta"
+certified: true 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Free"
+status-url: ""
+
+anchor-scheduling: true
+cron-scheduling: false
```

Comment:
This should be `true`

You can see if an integration has this feature (called Advanced Scheduling in the UI) by looking in it's **Settings** page:

1. Locate the Replication Frequency section.
2. Uncheck the **Use default** box.
3. Under the **Anchor Time** field, click the **Advanced** link.

If you see a bunch of fields (Minutes, Hours, etc) - the integration supports Advanced Scheduling, and this will be `true` :)

Reply:
```suggestion
cron-scheduling: true
```